tests/test_tradesimulation.py

```diff
@@ -17,92 +17,76 @@
 import pandas as pd
 from mock import patch
 
-from nose_parameterized import parameterized
 from six.moves import range
-from zipline import TradingAlgorithm
 from zipline.gens.sim_engine import BEFORE_TRADING_START_BAR
 
 from zipline.finance.asset_restrictions import NoRestrictions
 from zipline.finance import metrics
+from zipline.finance.trading import SimulationParameters
 from zipline.gens.tradesimulation import AlgorithmSimulator
 from zipline.sources.benchmark_source import BenchmarkSource
 from zipline.test_algorithms import NoopAlgorithm
-from zipline.testing.fixtures import (
-    WithDataPortal,
-    WithSimParams,
-    WithTradingEnvironment,
-    ZiplineTestCase,
-)
+import zipline.testing.fixtures as zf
 from zipline.utils import factory
-from zipline.testing.core import FakeDataPortal
+from zipline.testing.core import FakeDataPortal, parameter_space
 from zipline.utils.calendars.trading_calendar import days_at_time
 
 
-class BeforeTradingAlgorithm(TradingAlgorithm):
-    def __init__(self, *args, **kwargs):
-        self.before_trading_at = []
-        super(BeforeTradingAlgorithm, self).__init__(*args, **kwargs)
-
-    def before_trading_start(self, data):
-        self.before_trading_at.append(self.datetime)
-
-    def handle_data(self, data):
-        pass
-
-
-FREQUENCIES = {'daily': 0, 'minute': 1}  # daily is less frequent than minute
-
-
-class TestTradeSimulation(WithTradingEnvironment, ZiplineTestCase):
-
-    def fake_minutely_benchmark(self, dt):
-        return 0.01
-
-    def test_minutely_emissions_generate_performance_stats_for_last_day(self):
```

Comment:
I'm not sure what this test was originally supposed to test, but it looks like for a while it's just beeing testing the length of `sim_params.sessions` in a very confusing way...

Reply:
Going back in the logs this had originally been testing the value of `algo.perf_tracker.day_count`. `day_count` was removed with the risk rewrite, so I believe this test can be safely removed.